source/09-audit/05-inclusion-in-registry.rst

```diff
@@ -0,0 +1,16 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Application for inclusion in the COUNTER Registry
+-------------------------------------------------
+
+The Registry of report providers and their platforms for which COUNTER-compliant usage reports are available is maintained by COUNTER and publicly available at https://registry.projectcounter.org/.
+
+Report providers may apply to the Project Director (tasha.mellins-cohen@counterusage.org) for their products to be included on the Registry. Report providers will have to provide proof of initial compliance by including the results of COUNTER Validation Tool tests showing compliance for each reports, including testing both the upload of the tabular reports and SUSHI harvesting of the same report. Upon receipt of the application and proof of compliance, report providers MUST allow at least one of the COUNTER library test sites to evaluate their usage reports.
```

Comment:
same reports (plural)

proof of initial compliance (initial missing)

Here the term "library test sites" is used instead of "our listed library groups" (also without link or reference). The wording should be consistent.